example/new_test.go

```diff
@@ -1,4 +1,4 @@
-package index_test
+package esmini
```

Comment:
このファイルを example ディレクトリに入れてしまっているので、 esmini パッケージにはできないです。
（package example か package main のどちらかでしょう。

Reply:
こちら example ですので *_test.go と言うファイル名は一般的な命名規則に違反しています。

Reply:
パッケージ名はesminiではだめですね。。

ファイル名に関してですが、こちらのexampleのつもりで書いていました。

> Examples are tests
> Examples are compiled (and optionally executed) as part of a package's test suite.
> As with typical tests, examples are functions that reside in a package's _test.go files. Unlike normal test functions, though, example functions take no arguments and begin with the word Example instead of Test.
https://blog.golang.org/examples

Reply:
なるほどなるほど、勉強不足で全く理解できていませんでした。 🙇 

私の元々のイメージは、一応こんな感じでした。
https://github.com/kubernetes/client-go/blob/master/examples/create-update-delete-deployment/main.go

そのブログに例で挙げられている stringutil はどうやら現在パッケージ構成変わっちゃってますね。
https://github.com/golang/example/tree/master/stringutil

testing パッケージだと example でディレクトリを切るのではなく、_test.go に含めてしまっていますね。
https://golang.org/src/testing/benchmark_test.go

testing パッケージの方式でやると、ドキュメント吐いた時も Example という形で掲載されて良さげなので、一番良いのではないでしょうか？？

Reply:
testingパッケージの方式で進めたいと思います！

> testing パッケージだと example でディレクトリを切るのではなく、_test.go に含めてしまっていますね。
それぞれのテストに含めてしまう感じなんですね。
その方針に修正します！